### ISSUE TITLE
关于代码上的疑问

### ISSUE THREAD

**Issue by wushengcong** [Chang-Li-HFUT/FusionGAN#1]
作者你好，下载代码后，有关于代码的两个问题
1.文章中计算梯度的公式前面参数为8，但代码中是5。
self.g_loss_2=tf.reduce_mean(tf.square(self.fusion_image self.labels_ir))+5*tf.reduce_mean(tf.square(gradient(self.fusion_image) -gradient (self.labels_vi)))
2.文章中说判别器的第一层有使用填充补零，但代码中却是padding='VALID'。
conv1_vi=tf.nn.conv2d(img, weights, strides=[1,2,2,1], padding='VALID') + bias
